client/lib/api/who_date_format.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:intl/intl.dart';
+
+extension WHOFormat on DateTime {
```

Comment:
I guess you wanted to write an extension ;-) I would've included the `DateTime.fromMillisecondsSinceEpoch` step also as that's common to both usages.... and it's the msSinceEpoch that has midnight ambiguity. By the time it's converted to a date, it should be correct, so doing +1 in the format step doesn't fit so well to me.

I would've just extracted a single method and not bothered with a new file. We can move it to a separate file if something else uses it. I think though that this formatting may be specific to the WHO dashboard.... no idea if it extends to the 

```
String whoDashboardDateFormat(int epochMS) {
  var date = DateTime.fromMillisecondsSinceEpoch(epochMS);
  // Timestamps are for midnight so the day it applies to is ambiguous.
  // Add 1 day to match the WHO Dashboard: http://covid19.who.int
  date.add(Duration(days: 1))
  // Abbr to fit on single line: e.g. "Oct 18, 2020"
  return DateFormat.yMMMd().format(date);
}
```